skrub/_total_std_scaler.py

```diff
@@ -0,0 +1,102 @@
+import numpy as np
+
+
+def total_standard_deviation_scaler(X):
+    r"""Compute the total standard deviation scaler of X.
+
+    This scaler is used to normalize the vectors outputs of :class:`StringEncoder`,
+    :class:`TextEncoder` and :class:`GapEncoder`. It is computed during ``fit`` and
+    applied during ``transform``.
+
+    Conceptually, this scaler corresponds to the square root of the sum of the variances
+    of all columns of X.
+
+    Parameters
+    ----------
+    X : np.ndarray of shape (n_samples, n_features)
+        The matrix to compute the norm from.
+
+    Returns
+    -------
+    scaler : float,
```

Comment:
I think that this should be called "scale" and not "scaler". "scaler" gives me the impression that the value is an object that performs the scaling.

I think that this modification of the name should also be applied to the name of the attribute when it is store on an object: "self.scale_" and not "self.scaler_", or maybe "self.scaling_factor_".